pkg/convert/json_ngxtranslate_test.go

```diff
@@ -34,6 +34,20 @@ func Test_FromNgxTranslate(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:  "Message with placeholder",
```

Comment:
same, not sure if text in curly braces are considered placeholders

Maybe just:
```suggestion
			name:  "Message with special chars",
```

Reply:
I think `{world}` could be described as a parameter placeholder.